metabase/metabase
# 49664 add browser details in the json to display along with system info

Closes https://github.com/metabase/metabase/issues/49664

### Description

We want to collect browser information in the diagnostic window. This PR adds that capability using [bowser package](https://www.npmjs.com/package/bowser). 

### How to verify

Type cmd + f1 and download diagnostic data. You should see browserInfo section included in the json file.
